src/components/pages/donation_form/DonationSummary.vue

```diff
@@ -1,60 +1,63 @@
 <template>
-	<div class="form-summary-content">
-		<component
-			:is="addressTypeComponent"
-			:address="address"
-			:interval="interval"
-			:formatted-amount="formattedAmount"
-			:paymentType="paymentType"
-			:country="country"
-			:language-item="languageItem"
-			:salutation="salutation"
-		/>
+	<div class="form-summary-content" aria-live="polite">
+		<h3 id="donation-form-subheading" class="form-subtitle">{{ $t( 'donation_form_summary_title' ) }}</h3>
+		<p> {{ $t( 'donation_form_payment_headline' ) }}
+			{{ formattedAmount }}
+			<template v-if="interval"> {{ interval }}</template>
+			<template v-if="paymentType"> {{ $t('donation_summary_via') }} {{ paymentType }}</template>
+		</p>
+		<div class="switcher">
+			<div>
+				<DonorSummary :address="address" :countries="countries" :salutations="salutations" />
+			</div>
+			<div v-if="bankData.iban">
```

Comment:
If I select bank transfer and enter an IBAN then change to a different payment type it keeps the IBAN display in the summary. You can see it says PayPal and shows the IBAN box in the screenshot. You should check for the payment type as well as the existence of the IBAN.

![Screenshot 2025-06-03 at 08 51 54](https://github.com/user-attachments/assets/28084b3e-7465-40f9-8b82-034b6418a4c4)